packages/core/contracts/financial-templates/contract-for-difference/ContractForDifference.sol

```diff
@@ -100,10 +102,14 @@ contract ContractForDifference is Testable, Lockable {
      * @param _expirationTimestamp unix timestamp of when the contract will expire.
      * @param _collateralPerPair how many units of collateral are required to mint one pair of synthetic tokens.
      * @param _priceIdentifier registered in the DVM for the synthetic.
-     * @param _longTokenAddress ERC20 token used as long in the CFD. Requires mint and burn rights granted to this contract.
-     * @param _shortTokenAddress ERC20 token used as short in the CFD. Requires mint and burn rights granted to this contract.
+     * @param _longTokenAddress ERC20 token used as long in the CFD. Requires mint and burn rights granted to this
```

Comment:
feels like it would be better to just shorten this by 1 word rather than using a whole extra line. Can leave this for a subsequent cleanup PR though where we try to reduce any of these extra wrapped lines.

Reply:
Done!